.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: cpp
+compiler:
+  - clang
+notifications:
+  email: false
+env:
+  matrix:
+    - JULIAVERSION="juliareleases"
+    - JULIAVERSION="julianightlies"
+before_install:
+  - sudo add-apt-repository ppa:staticfloat/julia-deps -y
+  - sudo add-apt-repository ppa:staticfloat/${JULIAVERSION} -y
+  - sudo apt-get update -qq -y
+  - sudo apt-get install libpcre3-dev julia -y
+  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+script:
+  - julia --check-bounds=yes -e 'versioninfo(); Pkg.init(); Pkg.clone(pwd()); Pkg.build("QuBase"); Pkg.test("QuBase")'
```

Comment:
We can use the [simplified version of the .yml file](https://groups.google.com/forum/?nomobile=true#!searchin/julia-users/Travis/julia-users/BtCxh4k9hZA/ngUvxdxOxQ8J) now.